karapace/config.py

```diff
@@ -35,6 +35,8 @@
     "group_id": "schema-registry",
     "host": "127.0.0.1",
     "port": 8081,
+    "advertised_port": None,
+    "use_secured_proxy": False,
```

Comment:
nitpick: maybe name this `advertised_protocol`.
request: Could you please add an `advertised_host` too?

Reply:
Thanks for your reply @hackaugusto. Yeah, I can change the name to `advertised_protocol`. But for `advertised_host`, there is already a config called `advertised_hostname`, please see (https://github.com/aiven/karapace/blob/eb5a7720475b66de5011722db4799e3815fdacaa/karapace/config.py#L26). Is `advertised_host` still required if there is an `advertised_hostname`?

Reply:
Existing `advertised_hostname` is fine, but maybe sort those such that all three `advertised_` are together in the list. Also now `use_secure_proxy`was renamed to `advertised_protocol` its type and logic need to change from boolean True/False to a string (`http` by default)

Reply:
Thanks @tvainika, changes are done. Please review it again, thanks a lot.

Reply:
> But for advertised_host, there is already a config called advertised_hostname, please see 

My bad, thanks for pointing it out :)